package.json

```diff
@@ -57,9 +57,9 @@
     "@aries-framework/askar": "0.4.2",
     "@aries-framework/react-native": "0.4.2",
     "@aries-framework/indy-sdk-to-askar-migration": "0.4.2",
-    "@credo-ts/drpc": "patch:@credo-ts/drpc@npm%3A0.5.0-alpha.145#./.yarn/patches/@credo-ts-drpc-npm-0.5.0-alpha.145-4e1953647c.patch",
+    "@credo-ts/drpc": "patch:@credo-ts/drpc@npm%3A0.5.0-alpha.151#~/.yarn/patches/@credo-ts-drpc-npm-0.5.0-alpha.151-5bad8b38bd.patch",
```

Comment:
seems odd that the patch path changed from `./.yarn/...` (relative) to `~/.yarn/...` (relative to user directory?)

Reply:
hmm, ya good catch, I'll change it to the package.json directory